spec/controllers/admin/membership_submissions_controller_spec.rb

```diff
@@ -9,7 +9,89 @@
     end
   end
 
-  pending "GET #pending"
-  pending "POST #pending"
-  pending "POST #reject"
+  describe "GET #pending" do
+    describe "when pending submissions are zero" do
+      before do
+        allow(@controller).to receive(:require_admin!).and_return(true)
+        get :pending
+      end
+
+      it "redirects to admin_path" do
+        expect(response).to redirect_to(admin_path)
+      end
+
+      it "sets flash/info" do
+        expect(flash[:info]).to match(/submissions/i)
+      end
+    end
+
+    describe "when pending submissions are grater than zero" do
+      before do
+        @slack_membership_1 = create(:slack_membership_submission, email: 'johndoe@email.com')
+        @slack_membership_2 = create(:slack_membership_submission, email: 'johndoe2@email.com')
+        allow(@controller).to receive(:require_admin!).and_return(true)
+        get :pending
+      end
+
+      it "does not set flash/info and returns first membership submission" do
+        expect(flash[:info]).to be_blank
```

Comment:
We're combining multiple expectations here which while sometimes is reasonable I feel like in this case we should split up as they are two very different things.  I'm also not sure it's strictly necessary to test that the flash isn't set. It doesn't hurt, but I'm not sure it provides much value.